Gemfile.lock

```diff
@@ -563,7 +563,7 @@ GEM
     runger_config (5.2.0)
       activesupport (>= 7.1.2)
     runger_email_reply_trimmer (0.3.0)
-    runger_style (2.12.0)
+    runger_style (2.13.0)
```

Comment:
I pulled this update in here so that I can have an example description line that is more than 100 characters.